account-integrations/safe/src/lib/FCL_Webauthn.sol

```diff
@@ -76,13 +76,13 @@ library FCL_WebAuthn {
 
     /** @notice Modified from original Fresh Crypto Lib code to use memory instead of calldata */
     function checkSignature(
```

Comment:
Calldata also needed for `WebAuthn_format` in this file:
```solidity
    function WebAuthn_format(
        bytes calldata authenticatorData,
        bytes1 authenticatorDataFlagMask,
        bytes calldata clientData,
        bytes32 clientChallenge,
        uint256 clientChallengeDataOffset,
        uint256[2] calldata // rs
    ) internal pure returns (bytes32 result)
```

We should remove the new function I added to `FCL_elliptic.sol` at some point too. It's called `ecdsa_verify_memory` and I only I added that to use vars stored in memory.

Then we should call the original function instead in `checkSignature` in this file:
```solidity
// we should change this
bool result = FCL_Elliptic_ZZ.ecdsa_verify_memory(message, rs, Q);

// to this
bool result = FCL_Elliptic_ZZ.ecdsa_verify(message, rs, Q);
``

Reply:
Resolved by https://github.com/getwax/wax/pull/93 which is now merged